alacritty/src/cli.rs

```diff
@@ -277,6 +284,7 @@ impl Options {
         if config.debug.print_events {
             config.debug.log_level = max(config.debug.log_level, LevelFilter::Info);
         }
+        config.hold = self.hold;
```

Comment:
I think moving this under config.shell will be better for structure, so

```rust
config.shell = ...

config.hold = self.hold

config.window = ...
```
or without empty line between shell and hold, idk.

Reply:
I don't think this is related to shell, so I see little reason to move it. However I do think we should add an empty line above it, since it's also not related to `print_events`.

Reply:
I've suggested it because there is a giant `debug` section in the end, so I treat this `debug` block like a finishing one. So moving somewhere `hold` about it end emphasize with new line will make sense in my opinion. Feel free to ignore this comment.

Reply:
If the `debug` block is at the end or not shouldn't matter that much as long as it's not interrupted and cleanly separated from everything else I think. Before it would be fine too though.

Reply:
I might put it under line 263, under the command bit, just like it is for the CLI matching: https://github.com/jwilm/alacritty/pull/2787/files/8c501bcdcb2794eeb42928294dd1cdbf900ed691#diff-d72ceef0f1f8da7f6ac545eb5dbd0f63R236-R247